src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -166,15 +170,84 @@ export const getApplicationBuilder = () => {
     computed(() => []),
   );
 
-  let trayMenuItemsStateFake: TrayMenuItem[];
+  let commandId = 0;
+  const makeFakeMenuItem = (opts: MenuItemConstructorOptions, menu: Menu): MenuItem => {
```

Comment:
Let's keep our own interface here. This is now Electron's opinion how menu item looks and way too complicated. Our interface can be simpler. That also makes us less tightly coupled with Electron.

Reply:
I think I understand.